src/test/components/TransformShortcuts.test.js

```diff
@@ -105,21 +105,21 @@ function testTransformKeyboardShortcuts(setup: () => TestSetup) {
     });
   });
 
-  it('handles collapse direct recursion', () => {
+  it('handles collapse recursion', () => {
     const { pressKey, getTransform, expectedFuncIndex } = setup();
     pressKey({ key: 'r' });
     expect(getTransform()).toMatchObject({
-      type: 'collapse-direct-recursion',
+      type: 'collapse-recursion',
       funcIndex: expectedFuncIndex,
     });
   });
 
-  it('handles collapse indirect recursion', () => {
+  it('handles collapse direct recursion', () => {
     const { pressKey, getTransform, expectedFuncIndex } = setup();
     pressKey({ key: 'R' });
     expect(getTransform()).toMatchObject({
-      type: 'collapse-indirect-recursion',
-      funcIndex: expectedFuncIndex,
+      type: 'collapse-direct-recursion',
+      funcIndex: expectedFuncIndex, // ??? no implementation?
```

Comment:
`toMatchObject` matches partial objects

Reply:
Oh, hah, I forgot about this part. Thanks, that makes sense! I'll just remove the comment.